test/simplify_algebra_test.cpp

```diff
@@ -262,4 +262,127 @@ TEST_CASE(simplify_inner_broadcast)
     EXPECT(p1 == p2);
 }
 
+TEST_CASE(simplify_add_conv1)
+{
+    migraphx::program p;
+    auto x = p.add_parameter("x", {migraphx::shape::float_type, {1, 128, 28, 28}});
+    auto w =
+        p.add_literal(migraphx::generate_literal({migraphx::shape::float_type, {256, 128, 3, 3}}));
+    auto y = p.add_parameter("y", {migraphx::shape::float_type, {1, 128, 28, 28}});
+    auto v =
+        p.add_literal(migraphx::generate_literal({migraphx::shape::float_type, {256, 128, 3, 3}}));
+    auto conv1 = p.add_instruction(migraphx::op::convolution{}, x, w);
+    auto conv2 = p.add_instruction(migraphx::op::convolution{}, y, v);
+    auto sum   = p.add_instruction(migraphx::op::add{}, conv1, conv2);
+    p.add_instruction(pass_op{}, sum);
+    auto s = p.get_shape();
+    p.compile(simplify_algebra_target{});
+    EXPECT(s == p.get_shape());
+    EXPECT(std::count_if(
+               p.begin(), p.end(), [](auto&& ins) { return ins.name() == "convolution"; }) == 1);
+}
+
+TEST_CASE(simplify_add_conv2)
```

Comment:
can you rewrite the names of the test cases to clarify what each one is testing?

Reply:
Done